### DISCUSSION TITLE
How to make the console output into CSV format?

### DISCUSSION THREAD

**Discussion by bemevangelista** [prowler-cloud/prowler#2340] [Q&A] [answered]
Hi, in the previous version of prowler, the console output is the same as the file that will be written in the output folder if I choose CSV. I made a script before that I will I just get the results from the console output and just write it on a customer file inside a customer folder (-M csv > result/cis-output-date.csv). Is there a way that I could make the console output into CSV format? Thank you.

**Comment by jfagoagas** [comment, marked as answer]
Hi @bemevangelista, I'm not sure if I understand your question correctly but to generate a CSV output with Prowler v3 you just need to execute the following command:
```
prowler <cloud-provider> -M csv -F result/prowler_execution.csv
```

You can get more information about the different reporting ways in the Prowler documentation here https://docs.prowler.cloud/en/latest/tutorials/reporting.

Thanks for using Prowler 🚀
